### PR TITLE
Feature screens

### PR DESCRIPTION
**Pull request by rafaelje**
This PR work on this issues:

* Main Screen
closed: https://github.com/flyve-mdm/flyve-mdm-android-agent/issues/87

* Splash Screen
closed: https://github.com/flyve-mdm/flyve-mdm-android-agent/issues/82

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,69 +11,65 @@
...
 
-    <!--for location information-->
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+    <!-- for location information -->
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
 
     <application
         android:name=".FlyveMDMApp"
```
Comment (ajsb85) [inline app/src/main/AndroidManifest.xml]: Why `FlyveMDMApp` instead `MDMAgent`?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.45.0"
+
+default_platform :android
+
+platform :android do
+
+  desc "Alpha release app. Deploy a new version to the Google Play Store - Alpha channel"
```
Comment (ajsb85) [inline fastlane/Fastfile]: Beta release and Beta channel

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -35,5 +35,9 @@
     <string name="ERROR_GLPI_PARTIAL_UPDATE">Error</string>
     <string name="ERROR_GLPI_DELETE">Error</string>
     <string name="ERROR_GLPI_PARTIAL_DELETE">Error</string>
+    <string name="title_activity_main">MainActivity</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (ajsb85) [inline app/src/main/res/values/strings.xml]: Remove or change this placeholder text

**Comment by ajsb85** [comment]
This branch has conflicts that must be resolved

File: app/src/main/java/org/flyve/mdm/agent/MainActivity.java [hunk 4]
```diff
@@ -1,75 +1,53 @@
-/*
```
Comment (hectorerb) [inline app/src/main/java/org/flyve/mdm/agent/MainActivity.java]: @rafaelje where is license header?

File: fastlane/Fastfile [hunk 5]
```diff
@@ -35,7 +35,11 @@ platform :android do
     #gradle(task: "test")
 
   supply(
+<<<<<<< HEAD
+     track: 'beta',
+=======
```
Comment (hectorerb) [inline fastlane/Fastfile]: @rafaelje could you solve this conflict?

**Pull request merged**